docs/cluster-agent/HORIZONTAL_POD_AUTOSCALING.md

```diff
@@ -0,0 +1,262 @@
+
+At this point, you should be seeing:
+```
+PODS
+
+NAMESPACE     NAME                                     READY     STATUS    RESTARTS   AGE
+default       datadog-agent-4c5pp                      1/1       Running   0          14m
+default       datadog-agent-ww2da                      1/1       Running   0          14m
+default       datadog-agent-2qqd3                      1/1       Running   0          14m
+[...]
+default       datadog-cluster-agent-7b7f6d5547-cmdtc   1/1       Running   0          16m
+default       nginx-6757dd8769-5xzp2                   1/1       Running   0          3m
+
+```
+
+Now is time to create a Horizontal Pod Autoscaler manifest. If you take a look at /manifest/cluster-agent/hpa-example/hpa-manifest.yaml, you will see:
+- The HPA is configured to autoscale the Deployment called nginx
+- The maximum number of replicas created will be 5 and the minumum is 1
+- The metric used is `nginx.net.request_per_s` and the scope is `kube_container_name: nginx`. Note that this metric format corresponds to the Datadog one.
+
+Every 30 seconds (this can be configured) Kubernetes will query the DCA to get the value of this metric and will autoscale proportionally if necessary.
+For advanced use cases, it is possible to have several metrics in the same HPA, as you can see [here](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-multiple-metrics) the largest of the proposed value will be the one chosen.
+
+Now, let's create our nginx deployment:
+
+`kubectl apply -f manifests/cluster-agent/hpa-example/nginx.yaml`
+
+Then, let's apply the HPA
+
+`kubectl apply -f manifests/cluster-agent/hpa-example/hpa-manifest.yaml`
+
+### Stressing your service
+
+Now is time to see the magic happening!
+
+If you curl the IP of the nginx service as follows:
+`curl <nginx_svc>:8090/nginx_status`, you should receive an output like:
+
+```
+$ curl 192.168.254.216:8090/nginx_status
+
+Active connections: 1 
+server accepts handled requests
+ 1 1 1 
+Reading: 0 Writing: 1 Waiting: 0 
+```
+
+Behind the scene, the number of request per second also increased. 
+This metric is being collected by the node agent, as it autodiscovered the nginx pod through its annotations (for more information on how autodiscovery works, see our doc [here](https://docs.datadoghq.com/agent/autodiscovery/#template-source-kubernetes-pod-annotations)).
+Therefore, if you stress it, you will see the uptick in your Datadog app. 
+As you referenced this metric in your HPA manifest, the DCA is also pulling its latest value every 20 seconds.
+Then, as Kubernetes queries the DCA to get this value, it will notice that the number is going above the threshold and will autoscale accordingly.
+
+Let's do it!
+
+Just run `while true; do curl <nginx_svc>:8090/nginx_status; sleep 0.1; done`
+And you should soon see the number of requests per second spiking, going above 9 which is the threshold over which we want to autoscale out nginx boxes.
```

Comment:
assuming `out` is a typo for `our`—replace with `the`
also, the concept of "we" is odd, and some grammar:

...`going above 9, the threshold over which the nginx boxes autoscale.`